### PR TITLE
Script Loader: remove now unnecessary polyfill dependencies

### PR DESCRIPTION
**Pull request by swissspidy**
To-do:

* [x] Maybe remove `wp-polyfill-inert` too

Trac ticket: https://core.trac.wordpress.org/ticket/60962

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by felixarntz — APPROVED** [review]
Thanks @swissspidy!

**Review by adamsilverstein — APPROVED** [review]
Fantastic!

File: src/wp-includes/script-loader.php [hunk 1]
```diff
@@ -103,7 +103,7 @@ function wp_default_packages_vendor( $scripts ) {
 		'wp-polyfill-element-closest',
 		'wp-polyfill-object-fit',
 		'wp-polyfill-inert',
-		'wp-polyfill' => array( 'wp-polyfill-inert', 'regenerator-runtime' ),
+		'wp-polyfill' => array( 'regenerator-runtime' ),
```
Comment (adamsilverstein) [inline src/wp-includes/script-loader.php]: based on Riad's feedback, this can now be removed

**Comment by swissspidy** [comment]
Committed in https://core.trac.wordpress.org/changeset/57981

**Pull request closed without merging**